python/examples/llama-index/backend/app/settings.py

```diff
@@ -0,0 +1,42 @@
+import os
+from typing import Dict
+
+from llama_index.core.settings import Settings
+from llama_index.embeddings.openai import OpenAIEmbedding
+from llama_index.llms.openai import OpenAI
+
+
+def llm_config_from_env() -> Dict:
+    from llama_index.core.constants import DEFAULT_TEMPERATURE
+
+    model = os.getenv("MODEL", "gpt-3.5-turbo-0125")
```

Comment:
Shall we bump the default up to "gpt-4" or later?

Reply:
I won't be the best person to answer. My opinion, gpt-4.

Reply:
Let's roll with `gpt-4-turbo-preview`.